<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clinklist</class>
 <widget class="QMainWindow" name="clinklist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clinklist</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>570</width>
      <height>410</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>570</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDeleteValue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>66</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>值（int）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDelete">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>300</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDelete">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>275</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDeletePos">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>删除位置 ：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLocate">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLocate">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>440</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLocate">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>415</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelLocateValue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>66</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>值（int）：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultPos">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>结果位置 ：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeControl">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>速度调节 ：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>创建</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInsertValue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>66</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>值（int）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInsert">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>160</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsert">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>插入</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxInsert">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>135</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInsert">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>插入位置 ：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelState">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>状态提示 ：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditState">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>545</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRandomInsert5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>尾端随机5个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
